code/parse/sexp.cpp

```diff
@@ -33016,7 +33033,10 @@ SCP_vector<sexp_help_struct> Sexp_help = {
 		"\tBecomes true when a waypoint is hit that is before the last one hit, which "
 		"indicates they have flown a waypoint twice.\r\n\r\n"
 		"Returns a boolean value.  Takes no arguments." },
-
+	{ OP_SECONDS_TO_GOAL, "Seconds-to-goal (Number training operator)\r\n"
```

Comment:
Sexp help should be rewritten so that it's time instead of training and goal instead of waypoint.  Otherwise it looks good.

Reply:
"Sexp help should be rewritten so that it's time instead of training"

Er, so like "Time-to-goal (Number time operator)" or what?

Reply:
Update, I found the group of time operators and have made the  sexp help to "Time-to-goal (Time operator)" or similar.  See the commit in the new PR for details